pkg/manager/meta/reclaim_policy_manager.go

```diff
@@ -38,10 +41,26 @@ func NewReclaimPolicyManager(pvcLister corelisters.PersistentVolumeClaimLister,
 	}
 }
 
+// NewReclaimPolicyMonitorManager returns a *reclaimPolicyManager
+func NewReclaimPolicyMonitorManager(pvcLister corelisters.PersistentVolumeClaimLister,
+	pvLister corelisters.PersistentVolumeLister,
+	pvControl controller.PVControlInterface) monitor.MonitorManager {
+	return &reclaimPolicyManager{
+		pvcLister,
+		pvLister,
+		pvControl,
+	}
+}
+
 func (rpm *reclaimPolicyManager) Sync(tc *v1alpha1.TidbCluster) error {
-	ns := tc.GetNamespace()
-	instanceName := tc.GetInstanceName()
+	return rpm.sync(tc.GetNamespace(), tc.GetInstanceName(), tc.IsPVReclaimEnabled(), tc.Spec.PVReclaimPolicy, tc)
+}
+
+func (rpm *reclaimPolicyManager) SyncMonitor(tm *v1alpha1.TidbMonitor) error {
+	return rpm.sync(tm.GetNamespace(), tm.GetName(), true, tm.Spec.PVReclaimPolicy, tm)
```

Comment:
```suggestion
	return rpm.sync(tm.GetNamespace(), tm.GetName(), false, tm.Spec.PVReclaimPolicy, tm)
```

Reply:
If set to `true` and we set `label.AnnPVCDeferDeleting` to the monitor PVC later, the PVC will be removed after monitor scaling in.